ampd/src/tm_address.rs

```diff
@@ -3,32 +3,13 @@ use std::hash::{Hash as StdHash, Hasher};
 
 use cosmrs::crypto;
 use cosmrs::AccountId;
-use cosmwasm_std::Uint256;
 use ethers::types::{Address, H256};
 use serde::{Deserialize, Serialize};
 
 pub type EVMAddress = Address;
```

Comment:
But there are things like `EVMAddress` defined in this mod. Why call it `tm_address`?

Reply:
ok I missed that 🤦 
I'll think of a new name